Mathlib/Data/Nat/Squarefree.lean

```diff
@@ -25,15 +25,16 @@ open Finset
 
 namespace Nat
 
-theorem squarefree_iff_nodup_factors {n : ℕ} (h0 : n ≠ 0) : Squarefree n ↔ n.factors.Nodup := by
+theorem squarefree_iff_nodup_primeFactorsList {n : ℕ} (h0 : n ≠ 0) :
```

Comment:
I think we should add deprecated aliases for all of these, no?

Reply:
even if the `factors` appearing in the RHS is itself deprecated already?

Reply:
How will people using these lemmas know how to update their code? I think they are needed. The point is not to say that they are deprecated but to give people one click fixes.